apps/web/app/layout.tsx

```diff
@@ -13,12 +13,23 @@ export const metadata: Metadata = {
     template: `%s - ${siteConfig.name}`,
   },
   description: siteConfig.description,
-  themeColor: [
-    { media: "(prefers-color-scheme: light)", color: "white" },
-    { media: "(prefers-color-scheme: dark)", color: "black" },
-  ],
 };
 
+export function generateViewport(): Viewport {
```

Comment:
when using the arrow function syntax (or just `const viewport = `), Next.js/TS raised a false-positive error - for now, let's use this syntax